instrumentation/opentelemetry-instrumentation-flask/src/opentelemetry/instrumentation/flask/__init__.py

```diff
@@ -388,13 +392,21 @@ def _start_response(status, response_headers, *args, **kwargs):
             duration_attrs_old = otel_wsgi._parse_duration_attrs(
                 attributes, _HTTPStabilityMode.DEFAULT
             )
+
+            if wrapped_app_environ.get(_ENVIRON_REQUEST_ROUTE_KEY, None):
+                duration_attrs_old[SpanAttributes.HTTP_ROUTE] = wrapped_app_environ.get(_ENVIRON_REQUEST_ROUTE_KEY)
+
             duration_histogram_old.record(
                 max(round(duration_s * 1000), 0), duration_attrs_old
             )
         if duration_histogram_new:
             duration_attrs_new = otel_wsgi._parse_duration_attrs(
                 attributes, _HTTPStabilityMode.HTTP
             )
+
+            if wrapped_app_environ.get(_ENVIRON_REQUEST_ROUTE_KEY, None):
+                duration_attrs_new[SpanAttributes.HTTP_ROUTE] = wrapped_app_environ.get(_ENVIRON_REQUEST_ROUTE_KEY)
```

Comment:
Shouldn't we use the new semconv attribute here?

Reply:
What do you mean by new semconv? To change the variable?
It is mandatory to have the key value "http.route"